.storybook/preview-head.html

```diff
@@ -0,0 +1,6 @@
+<link
+  rel="stylesheet"
+  href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
```

Comment:
are we worried about linking to external sources? / maybe this might be something we'd host like the other fonts?

Reply:
i guess if this is just for the storybook, that's fine, and the site would link to font-awesome in its own way

Reply:
yeah, i just did the easiest / most straightforward path for loading icons for storybook right now. we're still deciding how we want to use icons (SVG vs webfont, which library to use, etc) so i set it up so that we can render icons for now, but we'll want to make this match however we end up loading icons for the Icon component